src/Adapter/MSTest.CoreAdapter/Execution/TestMethodInfo.cs

```diff
@@ -735,6 +751,11 @@ private object CreateTestClassInstance(TestResult result)
             return classInstance;
         }
 
+        private Exception NullReferenceException()
```

Comment:
Was this for testing purposes? Is it used anywhere?

Reply:
It's code intellisense that was too clever, reverting.